apis/python/src/tiledbsoma/io/_registration/ambient_label_mappings.py

```diff
@@ -233,21 +233,23 @@ def from_isolated_soma_experiment(
                     obs_map.update(dict(zip(obs_ids, soma_joinids)))
 
                 for measurement_name in exp.ms:
-                    exp.ms[measurement_name].var
+                    expvar = exp.ms[measurement_name].var
                     if "var" not in exp.ms[measurement_name]:
```

Comment:
Slightly cleaner to reference `exp.ms[measurement_name].var` after the `"var" in exp.ms[measurement_name]` check?

I see that it works either way, but the sequencing looks a little strange.

Reply:
Done -- thanks @ryan-williams ! :)